src/components/site/header/index.jsx

```diff
@@ -1,40 +1,59 @@
 import { Link } from 'gatsby';
 import PropTypes from 'prop-types';
 import React from 'react';
-import Toolbar from 'react-md/lib/Toolbars/Toolbar';
+import { Button, Toolbar } from 'react-md';
```

Comment:
I was originally going to do that, but decided to use the native import because of the use case on the page.

On a side note, trying to import our updated version of the button, but i'm getting an error: a render issue in the Header component.